angular/src/directives/navigation/ion-router-outlet.ts

```diff
@@ -55,6 +56,16 @@ export class IonRouterOutlet implements OnDestroy, OnInit {
 
   tabsPrefix: string | undefined;
 
+  /**
+   * @experimental
+   *
+   * The `EnvironmentInjector` provider instance from the parent component.
+   * Required for using standalone components with `ion-router-outlet`.
+   *
+   * Will be deprecated and removed when Angular 13 support is dropped.
```

Comment:
Do we need a tech debt ticket for this? (Ditto for `ion-tabs`)

Reply:
I'm indifferent since this is an experimental API that could be ripped out completely if either Angular doesn't proceed with standalone as part of the official public API or if don't think it is a good architectural fit with Ionic. 

I think once we consider it public API, having tech debt to maintain it could make sense. 

But we also don't have many experimental features, so if we want to follow our existing process like everything else, I'm happy to create tickets 👍